### PR TITLE
Feature/examples search filter

### PR DESCRIPTION
**Pull request by TomVanDenBroeck**
## PR Checklist

This PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: [Contributing guidelines](https://github.com/digipolisantwerp/acpaas-ui_angular/blob/master/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] A [changelog entry](https://github.com/digipolisantwerp/acpaas-ui_angular/blob/master/guidelines/CHANGELOG.md) was added

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [*] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/forms/examples/forms/pages/search-filter/search-filter.page.ts [hunk 1]
```diff
@@ -1,7 +1,60 @@
...
+
+@NgModule({
+	imports: [
+		SearchFilterModule,
+	]
+});
+
+export class AppModule {};`;
+
+	public searchfilterExampleJS1 = `public stuff: SearchFilterChoice[] = [{
```
Comment (lievenvg) [inline packages/forms/examples/forms/pages/search-filter/search-filter.page.ts]: You can add `import { SearchFilterChoice } from '@acpaas-ui/ngx-components/forms';` here.

File: packages/forms/src/lib/search-filter/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# @acpaas-ui/ngx-components/forms
+
+The search-filter is a JIRA-like filter component with remote search capabilities.
+
+## Usage
+
+```typescript
+import { SearchFilterModule } from '@acpaas-ui/ngx-components/search-filter'`;
```
Comment (lievenvg) [inline packages/forms/src/lib/search-filter/README.md]: `.../forms` 😉

File: packages/forms/src/lib/search-filter/README.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+Visit our [documentation site](https://acpaas-ui.digipolis.be/) for full how-to docs and guidelines
+
+### API
+
+| Name         | Default value | Description |
+| -----------  | ------ | -------------------------- |
+| `@Input() id: string;` | - | Field id. |
+| `@Input() name: string;` | - | Field name. |
+| `@Input() flyoutSize: enum;` | `FlyoutSize.Small` | Passed along to `auiFlyout`. |
```
Comment (lievenvg) [inline packages/forms/src/lib/search-filter/README.md]: Perhaps describe it a bit more. It's straight forward, but you can copy some content from the flyout.

File: packages/forms/src/lib/search-filter/README.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+| `@Input() flyoutAlign: string;` | `'left'` | Passed along to `auiFlyout`. |
```
Comment (lievenvg) [inline packages/forms/src/lib/search-filter/README.md]: Same here.

File: packages/forms/src/lib/search-filter/README.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
+# @acpaas-ui/ngx-components/forms
+
+The search-filter is a JIRA-like filter component with remote search capabilities.
```
Comment (TriangleJuice) [inline packages/forms/src/lib/search-filter/README.md]: `search filter` (without hyphen).

File: packages/forms/src/lib/search-filter/README.md (hunk 3 above)
Comment (TriangleJuice) [reply packages/forms/src/lib/search-filter/README.md]: Suggestion: maybe it's better to provide link to the appropriate documentation?

File: packages/forms/src/lib/search-filter/README.md (hunk 4 above)
Comment (TriangleJuice) [reply packages/forms/src/lib/search-filter/README.md]: Same here ;-)

File: packages/forms/src/lib/search-filter/README.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+| `@Input() flyoutSize: enum;` | `FlyoutSize.Small` | Passed along to `auiFlyout`. |
+| `@Input() flyoutAlign: string;` | `'left'` | Passed along to `auiFlyout`. |
+| `@Input() label: string;` | `'Filter'` | Field label. |
+| `@Input() labelDeselect: string;` | `'Alles deselecteren'` | Clear button text. |
+| `@Input() labelResults: string;` | `'Resultaten'` | Result list header text. |
+| `@Input() labelNoResults: string;` | `'Geen resultaten gevonden.'` | Text shown when no results are found. |
+| `@Input() choices: SearchFilterChoice[];` | - | Available choices. |
+| `@Input() remote: boolean;` | `false` | Enable remote searching. |
+| `@Input() placeholder: string;` | `'Zoeken'` | Search field placeholder text. |
+| `@Input() inputDelay: string;` | `'150'` | Delay the search callback on the input field. |
```
Comment (TriangleJuice) [inline packages/forms/src/lib/search-filter/README.md]: Is this really a string? Then we should list it in our technical debt document.

File: packages/notifications/src/lib/notifications/store/store.initial-state.ts [hunk 7]
```diff
@@ -1,4 +1,5 @@
 import { NotificationsState } from './store.types';
+import { NOTIFICATIONS_INITIAL_VALUE } from './notifications/notifications.initial-state';
```
Comment (TriangleJuice) [inline packages/notifications/src/lib/notifications/store/store.initial-state.ts]: Please be careful when using `git stash`!

File: packages/forms/src/lib/search-filter/README.md (hunk 6 above)
Comment (TomVanDenBroeck) [reply packages/forms/src/lib/search-filter/README.md]: no number when verifying in the new components.  Older readme had a wrong type defined.

File: styleguide/styles.scss [hunk 8]
```diff
@@ -47,39 +47,7 @@ main {
...
-	justify-content: space-between;
-	margin: 0 auto;
-	max-width: $screen-lg;
-	width: 100%;
-
-	@include at($screen-sm) {
-		padding: 0 $spacer;
-	}
-}
+a-button--navigation
```
Comment (lievenvg) [inline styleguide/styles.scss]: What's this? Leftover?

File: packages/notifications/src/lib/notifications/store/store.initial-state.ts (hunk 7 above)
Comment (TriangleJuice) [reply packages/notifications/src/lib/notifications/store/store.initial-state.ts]: Deze zit er nog steeds in?!

File: styleguide/styles.scss [hunk 9]
```diff
@@ -47,40 +47,6 @@ main {
 	}
 }
 
-.o-header__link {
```
Comment (TriangleJuice) [inline styleguide/styles.scss]: Waarom wordt dit allemaal verwijderd?

File: packages/notifications/src/lib/notifications/store/store.initial-state.ts (hunk 7 above)
Comment (TomVanDenBroeck) [reply packages/notifications/src/lib/notifications/store/store.initial-state.ts]: ik kan de NOTIFICATIONS_INITIAL_VALUE toch niet weg doen, wat dan is er een error dat deze niet kan gevonden worden.  wat is uw voorstel?

**Comment by TriangleJuice** [comment]
Er zit nog een bug in de search filter component zelf: als er geen resultaten zijn, dan wordt er een paragraaf getoond met een verkeerde class (`u-padding-xs`) op. De correcte class is `u-margin-xs`.

**Pull request merged**